dali/test/python/test_pipeline_segmentation.py

```diff
@@ -140,32 +146,26 @@ def check_bbox_random_crop_adjust_polygons(file_root, annotations_file, batch_si
             np.testing.assert_allclose(expected_out_vertices, out_vertices, rtol=1e-4)
 
             # Checking the conversion to absolute coordinates
-            h, w, c  = image_shape
-            expected_out_vertices_abs = np.copy(expected_out_vertices)
-            for v in range(expected_out_vertices_abs.shape[0]):
-                expected_out_vertices_abs[v, 0] = expected_out_vertices[v, 0] * w
-                expected_out_vertices_abs[v, 1] = expected_out_vertices[v, 1] * h
+            h, w, _ = image_shape
+            wh = np.array([w, h])
+            whwh = np.array([w, h, w, h])
+            expected_out_vertices_abs = expected_out_vertices * wh
             np.testing.assert_allclose(expected_out_vertices_abs, out_vertices_abs, rtol=1e-4)
 
             # Checking clamping of the relative coordinates
-            expected_out_vertices_clamped = np.copy(expected_out_vertices)
-            np.clip(expected_out_vertices_clamped, a_min=0.0, a_max=1.0)
-            np.testing.assert_allclose(expected_out_vertices_clamped, out_vertices, rtol=1e-4)
```

Comment:
Comparison of wrong value (out_vertices) against mistakenly unmodified expected values... It passed because of two errors cancelling each other.